### DISCUSSION TITLE
Icon only links

### DISCUSSION THREAD

**Discussion by SHELA** [JulianPrieber/littlelink-custom#246] [Feature requests]
Hello,
It's possible to have only icons instead have button with text?

like in screenshot. Maybe to make checkbox when selecting  Predefined Site

![image](https://user-images.githubusercontent.com/429667/205099283-85abbbf5-52b5-4d2e-80c6-c34ec51d640a.png)

**Comment by JulianPrieber** [comment]
I'll have to work out a concept that works for us, but we're definitely working on it.
